### PR TITLE
Add WFPC2 support for MVM processing

### PR DESCRIPTION
**Pull request by stsci-hack**
These changes update the MVM processing to treat WFPC2 exposures much like WFC3/IR exposures by defining both `coarse` and `fine` layers for WFPC2 layers.  The actual changes included here are:

- update poller file creation code to work with WFPC2 filter names
- add WFPC2 `pc` and `wf` (subarray) readout modes to list of observations which should generate both `coarse` and `fine` products when interpreting the poller file in poller_utils
- add diagnostic output to `cell_utils` to make confirming SkyCell identifications a little more obvious
- add default parameter files to `pars/hap_pars` for WFPC2 SVM WF (subarray) data as well as MVM processing.  

These changes were tested by using them to successfully generate both fine and coarse layers for skycell-p0074x17y02 using data from visit `u5b806`.

### CODE REVIEW

File: drizzlepac/haputils/poller_utils.py [hunk 1]
```diff
@@ -484,7 +484,9 @@ def parse_mvm_tree(det_tree, all_mvm_exposures, log_level):
                 # mvm prod_info = 'skycell_p1234_x01y01 wfc3 uvis f200lp all 2009 1 drz'
                 #
                 prod_list = prod_info.split(" ")
-                pscale = 'fine' if prod_list[2].upper() != 'IR' else 'coarse'
+                multi_scale = prod_list[2].upper() in ['IR', 'PC', 'WF']
+                pscale = 'fine' if not multi_scale else 'coarse'
+                # pscale = 'fine' if prod_list[2].upper() != 'IR' else 'coarse'
```
Comment (mdlpstsci) [inline drizzlepac/haputils/poller_utils.py:489]: Please clean up obsolete Line 489.

**Review by mdlpstsci — CHANGES_REQUESTED** [review]
Small change requested to remove obsolete code.

Also, the wfpc2_wf_index.json files are missing from the [svm|mvm]parameters directories.

**Comment by stsci-hack** [comment]
Config file index issues were fixed in 723afbd.

**Review by mdlpstsci — APPROVED** [review]
Looks ready to go!

**Pull request merged**